src/patternfly/components/OptionsMenu/docs/code.md

```diff
@@ -0,0 +1,26 @@
+## Accessibility
+
+*This section to be updated once the React implementation is complete.*
+
+| Attribute | Applied To | Outcome |
+| -- | -- | -- |
+| `role` or `aria` | `pf-c-options-menu` |  accessibility notes. |
+*Note:* The attribute `aria-selected="true"` should be set programmatically to the selected item(s).
+
+
+## Usage
+
+| Class | Applied To | Outcome |
+| -- | -- | -- |
+| `.pf-c-options-menu` | `<div>` |  Initiates a custom options menu. |
+| `.pf-c-options-menu__toggle` | `<button>` |  Initiates a custom toggle. |
```

Comment:
It can be a `div` when the whole toggle shouldn't be interactive, and there is an interactive element (or elements) inside. I didn't think that was the case here, but per Jenn's comment above, I think this is going to support being a `div`, too

Reply:
Oooh, the select should be using a `<button>`. I captured that in https://github.com/patternfly/patternfly-next/issues/1280